src/management/__generated/managers/ActionsManager.ts

```diff
@@ -0,0 +1,810 @@
+    );
+
+    return new runtime.JSONApiResponse(response);
+  }
+
+  /**
+   * Retrieve all of an action\'s versions. An action version is created whenever<br/>an action is deployed. An action version is immutable, once created.<br/>
+   * Get an action\'s versions
+   */
+  async getVersions(
+    requestParameters: GetActionVersionsRequest,
+    initOverrides?: RequestInit | runtime.InitOverrideFunction
+  ): Promise<GetActionVersions200Response> {
+    const response = await this.getVersionsRaw(requestParameters, initOverrides);
+    return await response.value();
+  }
+
+  /**
+   * Retrieve all actions.<br/>
+   * Get actions
+   * @throws {RequiredError}
+   * @memberof ActionsManager
+   */
+  async getAllRaw(
+    requestParameters: GetActionsRequest,
+    initOverrides?: RequestInit | runtime.InitOverrideFunction
+  ): Promise<runtime.ApiResponse<GetActions200Response>> {
+    const queryParameters: any = {};
+
+    if (requestParameters.triggerId !== undefined) {
+      queryParameters['triggerId'] = requestParameters.triggerId;
+    }
+
+    if (requestParameters.actionName !== undefined) {
+      queryParameters['actionName'] = requestParameters.actionName;
+    }
+
+    if (requestParameters.deployed !== undefined) {
+      queryParameters['deployed'] = requestParameters.deployed;
+    }
+
+    if (requestParameters.page !== undefined) {
+      queryParameters['page'] = requestParameters.page;
+    }
+
+    if (requestParameters.per_page !== undefined) {
+      queryParameters['per_page'] = requestParameters.per_page;
+    }
+
+    if (requestParameters.installed !== undefined) {
+      queryParameters['installed'] = requestParameters.installed;
+    }
+
+    const response = await this.request(
+      {
+        path: `/actions/actions`,
+        method: 'GET',
+        query: queryParameters,
+      },
+      initOverrides
+    );
+
+    return new runtime.JSONApiResponse(response);
+  }
+
+  /**
+   * Retrieve all actions.<br/>
+   * Get actions
+   */
+  async getAll(
+    requestParameters: GetActionsRequest = {},
+    initOverrides?: RequestInit | runtime.InitOverrideFunction
+  ): Promise<GetActions200Response> {
+    const response = await this.getAllRaw(requestParameters, initOverrides);
+    return await response.value();
+  }
+
+  /**
+   * Retrieve the actions that are bound to a trigger. Once an action is created and deployed, it must be<br/>attached (i.e. bound) to a trigger so that it will be executed as part of a flow. The list of actions returned<br/>reflects the order in which they will be executed during the appropriate flow.<br/>
+   * Get trigger bindings
+   * @throws {RequiredError}
+   * @memberof ActionsManager
+   */
+  async getTriggerBindingsRaw(
+    requestParameters: GetBindingsRequest,
+    initOverrides?: RequestInit | runtime.InitOverrideFunction
+  ): Promise<runtime.ApiResponse<GetBindings200Response>> {
+    if (requestParameters.triggerId === null || requestParameters.triggerId === undefined) {
```

Comment:
We could add helpers in the runtime to simplify these (easier to cover with tests too):

```js
async getTriggerBindingsRaw(
    requestParameters: GetBindingsRequest,
    initOverrides?: RequestInit | runtime.InitOverrideFunction
  ): Promise<runtime.ApiResponse<GetBindings200Response>> {

    // or these could just be one helper
    validateRequiredPathParams(requestParameters, ['triggerId'])
    const queryParameters: any = applyQueryParams(requestParameters, ['page', 'per_page']);

    ...
```